src/models/title-concept.model.ts

```diff
@@ -0,0 +1,17 @@
+import type { Media } from "./game-media.model";
+
+export interface GameConcept {
```

Comment:
```suggestion
export interface TitleConcept {
```

To match the file name.